src/crystal/system/win32/file.cr

```diff
@@ -205,7 +205,7 @@ module Crystal::System::File
 
   def self.rename(old_path : String, new_path : String) : Nil
     if LibC.MoveFileExW(to_windows_path(old_path), to_windows_path(new_path), LibC::MOVEFILE_REPLACE_EXISTING) == 0
-      raise ::File::Error.from_winerror("Error renaming file", file: old_path, other: new_path)
+      return ::File::Error.from_winerror("Error renaming file", file: old_path, other: new_path)
```

Comment:
Redundant return

```suggestion
      ::File::Error.from_winerror("Error renaming file", file: old_path, other: new_path)
```

Reply:
As described above. But i found that method return type shouldn't be `Nil`.